.gitignore

```diff
@@ -308,3 +308,7 @@ tests/example2.*
 
 # Client data
 paperqa/clients/client_data/retractions.csv
+
+# Benchmark for rag-qa-arena
+lotte/
```

Comment:
Instead of this, can we `.gitignore` a `data/` folder? Then we put the dataset in there